### PR TITLE
Stable 1.7.0 merge and release notes

### PR DESCRIPTION
**Pull request by joejulian**
**What type of PR is this?**

Release

**What this PR does/ why we need it**:

1.7.0 release

**Which issue(s) this PR fixes**:

no issue

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

* [x] The commit message explains the changes and why are needed
* [x] The documentation is updated where needed.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Release Notes
 
+## stable-1.15-1.7.0, stable-1.16-1.7.0
+
+* [prometheus]
+   * [CHANGE] Restrict api extension RBAC rules
+   * [BUGFIX] Fix statefulset crash loop on kubernetes ([#219](https://github.com/mesosphere/kubernetes-base-addons/pull/219), [@shaneutt](https://github.com/shaneutt))
+* [dex-k8s-authenticator]: Now supports a kubectl credentials plugin for automatically managing identity tokens. Instructions for downloading the plugin and configuring kubectl can be found at `https://<cluster-ip>/token/plugin`. ([#212](https://github.com/mesosphere/kubernetes-base-addons/pull/212), [@jr0d](https://github.com/jr0d))
```
Comment (samvantran) [inline RELEASE_NOTES.md:8]: I think this is duplicated from 1.6.0?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+* [dex]: support specifying root CA for LDAP connectors in Dex controller. ([#224](https://github.com/mesosphere/kubernetes-base-addons/pull/224), [@jieyu](https://github.com/jieyu))
+* [elasticsearch]: Fixes an issue that may cause the elasticsearch addon to fail to deploy. ([#206](https://github.com/mesosphere/kubernetes-base-addons/pull/206), [@branden](https://github.com/branden))
```
Comment (samvantran) [inline RELEASE_NOTES.md:10]: This too

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+* [cert-manager]: `usages` is no longer definable as part of `issuerRef`, instead it is a key on its own ([#196](https://github.com/mesosphere/kubernetes-base-addons/pull/196), [@sebbrandt87](https://github.com/sebbrandt87))
+* [velero]: bump velero to chart version 3.0.3, which includes velero-minio RELEASE.2020-04-10T03-34-42Z ([#215](https://github.com/mesosphere/kubernetes-base-addons/pull/215), [@jieyu](https://github.com/jieyu))
+* [dex-k8s-authenticator] added support for the konvoy credentials plugin ([#193](https://github.com/mesosphere/kubernetes-base-addons/pull/193), [@jr0d](https://github.com/jr0d))
+* \[velero\]: switch minio backend logging from plaintext to json ([#216](https://github.com/mesosphere/kubernetes-base-addons/pull/216), [@vespian](https://github.com/vespian))
```
Comment (joejulian) [inline RELEASE_NOTES.md:14]:
```suggestion
* [velero]: switch minio backend logging from plaintext to json ([#216](https://github.com/mesosphere/kubernetes-base-addons/pull/216), [@vespian](https://github.com/vespian))
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
 # Release Notes
 
+## stable-1.15-1.7.0, stable-1.16-1.7.0
+
+*  \[prometheus\]
+   * \[CHANGE\] Restrict api extension RBAC rules
+   * \[BUGFIX\] Fix statefulset crash loop on kubernetes ([#219](https://github.com/mesosphere/kubernetes-base-addons/pull/219), [@shaneutt](https://github.com/shaneutt))
+* \[dex\]: support specifying root CA for LDAP connectors in Dex controller. ([#224](https://github.com/mesosphere/kubernetes-base-addons/pull/224), [@jieyu](https://github.com/jieyu))
+* \[cert-manager\]: `usages` is no longer definable as part of `issuerRef`, instead it is a key on its own ([#196](https://github.com/mesosphere/kubernetes-base-addons/pull/196), [@sebbrandt87](https://github.com/sebbrandt87))
```
Comment (samvantran) [inline RELEASE_NOTES.md:9]: last one :)

**Pull request merged**